content/docs/traffic-management/destination-types/backends/_index.md

```diff
@@ -4,89 +4,34 @@ weight: 20
 prev: /docs/traffic-management/destination-types/kube-services
 ---
 
+Use a Backend resource to define a backing destination that you want {{< reuse "docs/snippets/product-name.md" >}} to route to. For more information, see the [Backend API docs](/docs/reference/api/upstream). 
 
-Use Backend resources to define a backing destination for a route that you want {{< reuse "docs/snippets/product-name.md" >}} to route to.
+## About
 
-Backends can be compared to a [cluster](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto) in Envoy terminology. Each Backend must define a type. Supported types include `static` and `kubernetes`. Each type is handled by a different plugin in {{< reuse "docs/snippets/product-name.md" >}}. For more information, see [Types](#types). 
+To help you think about Backends, consider how they relate to the underlying open source projects that you might be familiar with:
 
-Backends allow you to add additional configuration to instruct {{< reuse "docs/snippets/product-name.md" >}} how to handle the request to the backing destination. For example, you can define that the destination requires the requests to be sent with the HTTP/2 protocol or that you want requests to be load balanced by using a specific load balancing algorithm. To route to an Backend resource, you reference the Backend in the `backendRefs` section of your HTTPRoute, just like you do when routing to a Kubernetes service directly. For more information, see [Routing](#routing). 
+* Envoy: The {{< reuse "docs/snippets/product-name.md" >}} Backend can be compared to a [cluster](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/cluster/v3/cluster.proto) in Envoy terminology.
```

Comment:
wondering if we should take this out and just say that the Backend represents a backing destination that is external to the cluster and as such cannot be represented as a Kubernetes Service. Examples include a static website or an AWS Lambda function.